### PR TITLE
chore(deploy): use the new rolling strategy

### LINKED ISSUE #1: Use rolling app deployment feature in CF CLI
https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html

### PR DESCRIPTION
**Pull request by abdonrd**
Fix #1

### CODE REVIEW

**Comment by asudoh** [comment]
Looks promising, one quick question is if you have tried to run your change yourself? Thanks @abdonrd!

**Comment by abdonrd** [comment]
@asudoh I tested locally like this:

Screenshot 2020-10-14 at 16 41 21

Screenshot 2020-10-14 at 16 43 19

**Comment by abdonrd** [comment]
@asudoh yeah, I was trying to get the CI to work... Do you know what i'm doing wrong?

File: tests/deploy.test.js [hunk 1]
```diff
@@ -20,65 +20,24 @@ describe('Deploying a CF app to IBM Cloud', () => {
...
-      ['cf', 'install-plugin', 'blue-green-deploy', '-f', '-r', 'CF-Community'],
-      execOptions
-    );
-    expect(exec).toHaveBeenNthCalledWith(
-      3,
-      'ibmcloud',
-      ['cf', 'blue-green-deploy', 'cf-app-foo', '--delete-old-apps'],
-      execOptions
-    );
+    expect(exec).toHaveBeenNthCalledWith(3, 'ibmcloud', ['cf', 'push', 'cf-app-foo', '--strategy', 'rolling'], execOptions);
```
Comment (asudoh) [inline tests/deploy.test.js:23]: The 1st arg of `toHaveBeenNthCalledWith()` means "the nth call of the spy". Probably it explains the CI failure? I see two other occurrences of the same pattern.

```suggestion
    expect(exec).toHaveBeenNthCalledWith(3, 'ibmcloud', ['cf', 'push', 'cf-app-foo', '--strategy', 'rolling'], execOptions);
```

Comment (abdonrd) [reply tests/deploy.test.js]: Yeah, thanks! 🎉

**Comment by abdonrd** [comment]
Ready! It's already green!

How can we prove it now?

**Comment by asudoh** [comment]
> How can we test it now?

Thank you for asking @abdonrd! The test should be creating a pseudo repo that integrates the action and pushes to a pseudo CF app. Note that you can use a commit hash as the GH action version.

**Comment by abdonrd** [comment]
@asudoh seems that it works:

https://github.com/IBM/pwa-lit-template/runs/1257059707?check_suite_focus=true#step:9:224

**Comment by asudoh** [comment]
Super thank you so much for testing @abdonrd! 🎉

**Pull request merged**